### PR TITLE
Added featured insight block, featured report block, and insight archive

### PR DESCRIPTION
**Pull request by justinvandenberg**
Changes:
* Added `hero-archive-insight` block
* Added `featured-report` block
* Added `featured-insight` block
* Added `center` to resized images

Screenshots:
Screen Shot 2019-10-29 at 10 38 06 AM
Screen Shot 2019-10-29 at 10 38 19 AM
Screen Shot 2019-10-29 at 10 38 31 AM
Screen Shot 2019-10-29 at 10 38 44 AM
Screen Shot 2019-10-29 at 10 39 04 AM
Screen Shot 2019-10-29 at 10 39 22 AM
Screen Shot 2019-10-29 at 10 39 33 AM
Screen Shot 2019-10-29 at 10 39 43 AM

Time tracking:
[NPM] -> [01] Redesign 
If needed, Grace can add you.

### CODE REVIEW

File: web/themes/npm/views/_blocks/featured-report/index.twig [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
+            </div>
+          {% endif %}
 
-            {% if featured_work.text %}
-              <p class="text text-14">
-                {{ featured_work.text }}
-              </p>
-            {% endif %}
+          <a class="button-outline-white mt-25 lg:mt-30" href="{{ report.button_link ?? report.link }}">
+            {{ report.button_text ?? 'Read More' }}
```
Comment (mzlock) [inline web/themes/npm/views/_blocks/featured-report/index.twig]: Can you improve that `Read More` text? Like: `Read More <span class="sr-only">about {{ report.title }}</span>`

File: web/themes/npm/views/_blocks/hero-archive-insight/index.twig [hunk 2]
```diff
@@ -1,48 +1,51 @@
...
+          </div>
 
-            {% if block.heading %}
-              <h1 class="heading-3">
-                {{ block.heading }}
-              </h1>
-            {% endif %}
+          <h1 class="heading-3">
+            {{ insight_report.title }}
+          </h1>
```
Comment (mzlock) [inline web/themes/npm/views/_blocks/hero-archive-insight/index.twig]: Is an `<h1>` correct for a heading inside a block?

File: web/themes/npm/views/_blocks/insights/insight.twig [hunk 3]
```diff
@@ -1,29 +1,23 @@
...
-        <li class="leading-none mb-10 px-10">
-          {{ insight.post_date }}
-        </li>
-      {% endif %}
-    </ul>
-  {% endif %}
+    <li class="leading-none mb-10 px-10">
+      {{ insight.post_date | date('F Y') }}
+    </li>
+  </ul>
```
Comment (mzlock) [inline web/themes/npm/views/_blocks/insights/insight.twig]: Same here. I'm not sure if this should be a list??

File: web/themes/npm/app/post-types.php [hunk 4]
```diff
@@ -65,6 +65,16 @@ function npm_register_post_types()
         )
     );
 
+    register_post_type('report',
+        array(
+            'labels' => npm_create_post_type_labels('Reports', 'Report'),
+            'has_archive' => false,
+            'menu_icon' => '',
```
[the remark this answers is not in the document]
Comment (justinvandenberg) [reply web/themes/npm/app/post-types.php]: I was actually planning on bringing in some external for these! Just haven't gotten around to it yet.

File: web/themes/npm/views/_blocks/featured-insight/index.twig [hunk 5]
```diff
@@ -1,41 +1,46 @@
...
+            </li>
 
-            {% if featured_image %}
-              <img src="{{ featured_image.src | resize(1440, 900) }}" alt="{{ featured_image.alt }}" />
-            {% endif %}
-          </div>
+            <li class="leading-none mb-10 px-10">
+              {{ insight.post_date | date('F Y') }}
+            </li>
+          </ul>
```
[the remark this answers is not in the document]
Comment (justinvandenberg) [reply web/themes/npm/views/_blocks/featured-insight/index.twig]: I was on the fence about it too, I'll swap that out.

File: web/themes/npm/views/_blocks/hero-archive-insight/index.twig (hunk 2 above)
Comment (justinvandenberg) [reply web/themes/npm/views/_blocks/hero-archive-insight/index.twig]: I hope they always use one of the `hero` blocks first, but there is the possibility they don't. What do you think I should do?

Comment (mzlock) [reply web/themes/npm/views/_blocks/hero-archive-insight/index.twig]: I suppose there are two options:

- make it an H1 only when this block is first - so you'd have to pass in the index for the block loop
- make this always an `<h2>` and have a `<h1 class="sr-only">` as the _real_ page title at the top

File: web/themes/npm/views/_blocks/featured-report/index.twig [hunk 6]
```diff
@@ -1,41 +1,36 @@
...
-            </a>
-          </div>
+          {% if image.src %}
+            <div class="-mx-20 lg:mx-0">
+              <img src="{{ image.src | resize(1440, 1080, 'center') }}" alt="{{ image.alt }}" />
+            </div>
+          {% endif %}
         </div>
-      {% endif %}
+      </div>
```
Comment (olets) [inline web/themes/npm/views/_blocks/featured-report/index.twig]: looks like this template is very similar to web/themes/npm/views/_blocks/featured-insight/index.twig. I didn't go through it line by line but maybe something there? understand it wouldn't be in the scope of this pr

**Pull request closed without merging**